lib/hanami/assets/configuration.rb

```diff
@@ -120,6 +124,22 @@ def digest(value = nil)
         end
       end
 
+      # Subresource integrity mode
+      #
+      # Determine if the helpers should generate the integrity attribute for an
+      # asset. Usually this is turned on in production mode.
+      #
+      # @since 0.3.0-add-options-to-javascript-helper
+      def subresource_integrity(*values)
+        if values.empty?
+          @subresource_integrity
+        elsif values.length == 1
+          @subresource_integrity = values.first
+        else
+          @subresource_integrity = values
+        end
+      end
```

Comment:
You can see in the tests, but the reason I do this is to allow the following values:
- `nil` (when accessing value)
- `true` (when using default)
- `:sha256 (for a single algorithm)
- `:sha256, :sha512` (a bracket-less array)
- `[:sha256, :sha512]` (a bracketed array)

I only had bracketed arrays to begin (which let me avoid doing the splatting) but bracket-less arrays are really nice.